services/cannon/src/Cannon/API.hs

```diff
@@ -70,11 +69,8 @@ sitemap = do
     head "/i/status" (continue (const $ return empty)) true
 
 monitoring :: Media "application" "json" -> Cannon Response
-monitoring = const $ do
-    m <- monitor
-    s <- D.size =<< clients
-    gaugeSet (fromIntegral s) (path "net.websocket.clients") m
-    json <$> Metrics.render m
+monitoring _ = do
```

Comment:
The `do` can be removed? :P

Reply:
i was tempted to argue against that, but fine...  :)